affiliatewp-affiliate-area-shortcodes.php

```diff
@@ -22,242 +22,131 @@
-			$locale   = apply_filters( 'plugin_locale',  get_locale(), 'affiliatewp-affiliate-area-shortcodes' );
-			$mofile   = sprintf( '%1$s-%2$s.mo', 'affiliatewp-affiliate-area-shortcodes', $locale );
+		// Maybe hook-in the bootstrapper.
+		if ( class_exists( 'AffiliateWP_Affiliate_Area_Shortcodes' ) ) {
 
-			// Setup paths to current locale file
-			$mofile_local  = $lang_dir . $mofile;
-			$mofile_global = WP_LANG_DIR . '/affiliatewp-affiliate-area-shortcodes/' . $mofile;
+			// Bootstrap to plugins_loaded.
+			$affwp_version = get_option( 'affwp_version' );
 
-			if ( file_exists( $mofile_global ) ) {
-				// Look in global /wp-content/languages/affiliatewp-affiliate-area-shortcodes/ folder
-				load_textdomain( 'affiliatewp-affiliate-area-shortcodes', $mofile_global );
-			} elseif ( file_exists( $mofile_local ) ) {
-				// Look in local /wp-content/plugins/affiliatewp-affiliate-area-shortcodes/languages/ folder
-				load_textdomain( 'affiliatewp-affiliate-area-shortcodes', $mofile_local );
+			if ( version_compare( $affwp_version, '2.7', '>=' ) ) {
+					add_action( 'affwp_plugins_loaded', array( $this, 'bootstrap' ), 100 );
 			} else {
-				// Load the default language files
-				load_plugin_textdomain( 'affiliatewp-affiliate-area-shortcodes', false, $lang_dir );
+					add_action( 'plugins_loaded', array( $this, 'bootstrap' ), 100 );
 			}
-		}
-
-		/**
-		 * Include necessary files
-		 *
-		 * @access      private
-		 * @since       1.1
-		 * @return      void
-		 */
-		private function includes() {
-			require_once AFFWP_AAS_PLUGIN_DIR . 'includes/class-shortcodes.php';
-		}
 
-		/**
-		 * Setup the default hooks and actions
-		 *
-		 * @since 1.1
-		 *
-		 * @return void
-		 */
-		private function hooks() {
-			// plugin meta
-			add_filter( 'plugin_row_meta', array( $this, 'plugin_meta' ), null, 2 );
+			// Register the activation hook.
+			register_activation_hook( __FILE__, array( $this, 'install' ) );
 		}
+	}
 
-		/**
-		 * Modify plugin metalinks
-		 *
-		 * @access      public
-		 * @since       1.1
-		 * @param       array $links The current links array
-		 * @param       string $file A specific plugin table entry
-		 * @return      array $links The modified links array
-		 */
-		public function plugin_meta( $links, $file ) {
-			if ( $file == plugin_basename( __FILE__ ) ) {
-				$plugins_link = array(
-				'<a title="' . __( 'Get more add-ons for AffiliateWP', 'affiliatewp-affiliate-area-shortcodes' ) . '" href="'. admin_url( 'admin.php?page=affiliate-wp-add-ons' ) . '">' . __( 'More add-ons', 'affiliatewp-affiliate-area-shortcodes' ) . '</a>'
-				);
-
-				$links = array_merge( $links, $plugins_link );
-			}
+	/**
+	 * Install, usually on an activation hook.
+	 *
+	 * @since 1.2
+	 */
+	public function install() {
+		// Bootstrap to include all of the necessary files.
+		$this->bootstrap();
 
-			return $links;
+		if ( defined( 'AFFWP_AAS_VERSION' ) ) {
+			update_option( 'affwp_ass_version', AFFWP_AAS_VERSION );
```

Comment:
I think this was supposed to be `affwp_aas_version` 😆 
```suggestion
			update_option( 'affwp_aas_version', AFFWP_AAS_VERSION );
```